lib/modules/manager/index.ts

```diff
@@ -78,10 +78,13 @@ export function extractPackageFile(
 
 export function getRangeStrategy(config: RangeConfig): RangeStrategy | null {
   const { manager, rangeStrategy } = config;
-  if (!manager || !managers.has(manager)) {
+  if (!manager) {
+    return null;
+  }
+  const m = managers.get(manager)! ?? customManagers.get(manager)!;
```

Comment:
```suggestion
  const m = managers.get(manager) ?? customManagers.get(manager);
```
non-null cast not needed here, you've a null check below.